statistics/handle/update.go

```diff
@@ -549,7 +549,7 @@ func (h *Handle) dumpTableStatCountToKV(id int64, delta variable.TableDelta) (up
 	startTS := txn.StartTS()
 	updateStatsMeta := func(id int64) error {
 		var err error
-		if h.IsTableLocked(id) {
+		if h.IsTableLocked(id, false) {
```

Comment:
I think not locking is because it is already locked in previous code and Mutex is not a reentrant lock? If so, could you add some comments here and above the method defination?

Reply:
Mutex is not a reentrant lock. I have enabled deadlock in the ```Handle```. so that if I set ```lock``` true, we will meet deadlock.